deploy/docker/hail_search/sync_datasets

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
I've pushed the `initContainers` logic into this script to help with the environment variable looping piece.  To me, this is _right_ on the line of needing to be written in python.